web/next.config.js

```diff
@@ -14,7 +14,7 @@ const withPWA = require('next-pwa')({
 });
 
 const nextConfig = withPWA({
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
Just want to keep a note here, did we encountered any blockers resulting us turning this off?

Reply:
yes, using strict mode cause the `useEffect` hook to run twice, resulting in some weird behaviour when using the authToken to get acessToken